SimPEG/Directives.py

```diff
@@ -239,14 +239,51 @@ def endIter(self):
 
 
 
-# class UpdateReferenceModel(Parameter):
-
-#     mref0 = None
-
-#     def nextIter(self):
-#         mref = getattr(self, 'm_prev', None)
-#         if mref is None:
-#             if self.debug: print 'UpdateReferenceModel is using mref0'
-#             mref = self.mref0
-#         self.m_prev = self.invProb.m_current
-#         return mref
+class update_IRLS(InversionDirective):
```

Comment:
Can we call it `UpdateReferenceModel`? right now it is `update_IRLS` is that not correct?